pkg/csi/service/vanilla/controller.go

```diff
@@ -720,7 +720,7 @@ func (c *controller) ControllerUnpublishVolume(ctx context.Context, req *csi.Con
 	if err != nil {
 		msg := fmt.Sprintf("Validation for UnpublishVolume Request: %+v has failed. Error: %v", *req, err)
 		log.Error(msg)
```

Comment:
`validateVanillaControllerUnpublishVolumeRequest` returns an error code. I guess we can just change this to `return nil, err` instead.